properties.schema

```diff
@@ -19,6 +19,14 @@
       "inputType": "Text",
       "validators": [],
       "translatable": true
+    },
+      "ariaPinLabel": {
+      "type": "string",
+      "required": true,
+      "default": "Hotgraphic pin number ",
```

Comment:
Is a learner going to know what a 'hotgraphic' is? "Clickable pin" or "Selectable pin" might be better and then gives the learner a strong indication of what it does.